go.mod

```diff
@@ -216,6 +244,7 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/apiserver v0.33.2
```

Comment:
can you run `go mod tidy`. The dependencies are not well sorted. This one for example should go at the top of the file with the other direct dependencies